### DISCUSSION TITLE
Failure Modes, Reliability (field experience)

### DISCUSSION THREAD

**Discussion by NitramH** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#570] [Ideas]
Dear all,

after about 2years operation the first reliability issues in the control appeared (beside leaking connections of the spa due to corrosive seals). 

Due to the fact that 3 different systems are interacting, failure investigation is like hell.
Possible root causes in my system:
-homeassistant upgrade issue (excluded by restore backup)
-miami layzspa electonics (replaced base but kept the display part, next step to replace display as well)
-level converter electronics (not the root cause)
-cable and connectors (not the root cause)
-d1 mini electronics (not the root cause)

In my special case the control is delayed up to 10s, the result jitters randomly or just falls back to original value.
To find the root cause I invested a lot of time, but did not found it yet. Therefore here is my idea which I would like to post.
What do you think about a chapter where users can describe the typical weak points of this system to help others to 
find root causes faster? E.g. by describing the failure picture and a possible root cause in table form where everyone can edit it?

BR,
Martin

**Comment by visualapproach** [comment]
You mean in the manual? Or a wiki? It would be great but it also would need someone creating it and managing it... Feel free to dive in :-)
